endoreg_db/models/media/video/video_file_io.py

```
@@ -20,9 +20,37 @@ def _get_raw_file_path(video: "VideoFile") -> Optional[Path]:
+                # Check if file is in sensitive subdirectory
+                sensitive_path = data_paths["video"] / "sensitive" / rel_name
+                if sensitive_path.exists():
+                    return sensitive_path.resolve()
+                
+                # Check direct raw_file.path if available
+                try:
+                    direct_path = Path(video.raw_file.path)
+                    if direct_path.exists():
+                        return direct_path.resolve()
+                except Exception:
+                    pass  # Fallback to original behavior
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)